src/DynamoCoreWpf/ViewModels/Watch3D/Shaders/Material.fx

```diff
@@ -27,7 +27,8 @@ float2 vShadowMapSize = float2(1024, 1024);
 float4 vShadowMapInfo = float4(0.005, 1.0, 0.5, 0.0);
 
 float4 vSelectionColor = float4(0.0, 0.62, 1.0, 1.0);
-
+float4 vTransparentMaterial = 0.1f;
+float4 vTransparentLinePoint = 0.2f;
```

Comment:
Does this have the same effect as the following?

`float4 vTransparentLinePoint = float4(0.2f, 0.2f, 0.2f, 0.2f);`